Fix failing ARM64 benchmarks upload

```log
Global total time: 00:14:02 (842.44 sec), executed benchmarks: 748
// * Artifacts cleanup *
$ popd
$ pushd "/ssd/dotnetbuild/work/3d939876-3fdc-4ae4-a5c1-b26bd3c55872/Work/97375e06-746d-4617-9c30-836b641e052e/Exec/bin"
$ /usr/bin/python3.5 /ssd/dotnetbuild/work/3d939876-3fdc-4ae4-a5c1-b26bd3c55872/tools/Microsoft.BenchView.JSONFormat/tools/submission-metadata.py --name ".NET Performance - private - merge 305" --user-email dotnet-bot@microsoft.com
[2019-02-19 22:33][INFO ] Serialized object written to: "submission-metadata.json"
$ popd
$ pushd "/ssd/dotnetbuild/work/3d939876-3fdc-4ae4-a5c1-b26bd3c55872/Work/97375e06-746d-4617-9c30-836b641e052e/Exec/bin"
$ /usr/bin/python3.5 /ssd/dotnetbuild/work/3d939876-3fdc-4ae4-a5c1-b26bd3c55872/tools/Microsoft.BenchView.JSONFormat/tools/machinedata.py
[2019-02-19 22:33][ERROR] <class 'ValueError'>: Manufacturer is unknown and must be supplied by end user.
[2019-02-19 22:33][ERROR] Traceback (most recent call last):
  File "/ssd/dotnetbuild/work/3d939876-3fdc-4ae4-a5c1-b26bd3c55872/tools/Microsoft.BenchView.JSONFormat/tools/machinedata.py", line 273, in main
    raise ValueError("Manufacturer is unknown and must be supplied by end user.")
ValueError: Manufacturer is unknown and must be supplied by end user.
```

--- a/scripts/benchmarks_ci.py
+++ b/scripts/benchmarks_ci.py
@@ -364,8 +364,12 @@ def __run_benchview_scripts(
         working_directory=bin_directory,
         name=submission_name)
 
+    machine_manufacturer = 'Unknown' if args.architecture == 'arm64' else ''
+
     # BenchView machinedata.py
-    benchviewpy.machinedata(working_directory=bin_directory)
+    benchviewpy.machinedata(
+        working_directory=bin_directory,
+        machine_manufacturer=machine_manufacturer)
 
     for framework in args.frameworks:
         target_framework_moniker = micro_benchmarks \

--- a/scripts/benchview.py
+++ b/scripts/benchview.py
@@ -104,7 +104,7 @@ class BenchView:
             self.python, path.join(self.tools_directory, 'machinedata.py')
         ]
         if machine_manufacturer:
-            cmdline += ['--machine_manufacturer', machine_manufacturer]
+            cmdline += ['--machine-manufacturer', machine_manufacturer]
         RunCommand(cmdline, verbose=self.verbose).run(working_directory)
 
     def measurement(self, working_directory: str) -> None:

--- a/scripts/benchview.py
+++ b/scripts/benchview.py
@@ -97,12 +97,14 @@ class BenchView:
             cmdline += ['--source-timestamp', source_timestamp]
         RunCommand(cmdline, verbose=self.verbose).run(working_directory)
 
-    def machinedata(self, working_directory: str) -> None:
+    def machinedata(self, working_directory: str, machine_manufacturer: str) -> None:
         '''Wrapper around BenchView's machinedata.py'''
 
         cmdline = [
             self.python, path.join(self.tools_directory, 'machinedata.py')
         ]
+        if machine_manufacturer:
+            cmdline += ['--machine_manufacturer', machine_manufacturer]
         RunCommand(cmdline, verbose=self.verbose).run(working_directory)
 
     def measurement(self, working_directory: str) -> None:

